crowbar_framework/config/puma.rb

```diff
@@ -52,6 +52,17 @@
   FileUtils.mkdir_p File.join(ROOT, name)
 end
 
+CROWBAR_LIB_DIR = "/opt/dell/crowbar_framework/lib".freeze
+$LOAD_PATH.push CROWBAR_LIB_DIR if Dir.exist?(CROWBAR_LIB_DIR)
+
+# set the end_step status during the upgrade
+if File.exist?("/var/lib/crowbar/upgrade/7-to-8-upgrade-running")
```

Comment:
Do we need a shell pattern here as you did in a previous change?

Reply:
Nope, this code is running on SOC8 and we're only interesting in SOC8 part of the upgrade